src/styles/blocks/card.css

```diff
@@ -0,0 +1,92 @@
+.card {
+  display: flex;
+  flex-direction: column;
+
+  padding: 16px;
+  width: 200px;
+  height: 408px;
+  border-radius: 5px;
+  background-color: #fff;
+  border: 1px solid #f3f3f3;
+}
+
+.card__img {
+  width: 160px;
+  height: 134px;
+  margin: 16px auto 40px;
+}
+
+.card__title {
+  font-size: 12px;
+  font-weight: 500;
+  color: #060b35;
+  line-height: 18px;
+  margin-bottom: 4px;
+}
+
+.card__code {
+  font-size: 10px;
+  font-weight: 400;
+  color: #616070;
+  height: 14px;
+  line-height: 14px;
+  margin-bottom: 16px;
+}
+
+.card__rate {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  margin-bottom: 24px;
+}
+
+.card__rate-reviews {
+  font-size: 10px;
+  font-weight: 400;
+  color: #060b35;
+}
+
+.card__price {
+  display: flex;
+  justify-content: space-between;
+  margin-bottom: 16px;
+  color: #616070;
+  line-height: 18px;
+}
+
+.card__price-text {
+  font-weight: 400;
+  font-size: 12px;
+  color: #616070;
+}
+
+.card__price-value {
+  font-weight: 700;
+  font-size: 16px;
+  color: #060b35;
+}
+
+.card__button {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.card__link {
+  font-size: 14px;
+  font-weight: 700;
+  color: #fff;
+  text-decoration: none;
+  text-transform: uppercase;
+  background-color: #00acdc;
+  border: 1px solid transparent;
+  border-radius: 5px;
+  padding: 12px 55px;
+}
+
+.card__link:hover {
+  background-color: #fff;
+  color: #00acdc;
+  border: 1px solid #00acdc;
+  transition-duration: 300ms;
```

Comment:
should add transition-duration for element styles (not on hover). If you add duration on hover, your styles add smoothly but remove fast. Also, need always add transition property